lib/salus/scanners/report_gradle_deps.rb

```diff
@@ -0,0 +1,46 @@
+require 'salus/scanners/base'
+
+# Report Java library usage
+
+module Salus::Scanners
+  class ReportGradleDeps < Base
+    UNKNOWN_VERSION = ''.freeze
```

Comment:
It seems like multiple classes use `UNKNOWN_VERSION`. So it would be better to move `UNKNOWN_VERSION` to the Base class.